lib/srv/reexec.go

```diff
@@ -317,6 +338,18 @@ func RunCommand() (errw io.Writer, code int, err error) {
 		pamEnvironment = pamContext.Environment()
 	}
 
+	// Alert the parent process that the child process has completed any setup operations,
+	// and that we are now waiting for the continue signal before proceeding. This is needed
+	// to ensure that PAM changing the cgroup doesn't bypass enhanced recording.
+	ready = true
+	if _, err := readyfd.Write([]byte{0}); err != nil {
```

Comment:
Since this is a different fd, do we need to even write anything to it, or can we just close it to unblock? Value is being ignored after reading anyways.